newIDE/app/src/SceneEditor/index.js

```diff
@@ -1281,6 +1281,33 @@ export default class SceneEditor extends React.Component<Props, State> {
             onClose={() => this.editLayoutVariables(false)}
           />
         )}
+        <InfoBar
+          show={this.state.objectsListAdditionalServiceInfoBar.show}
+          identifier={this.state.objectsListAdditionalServiceInfoBar.identifier}
+          message={this.state.objectsListAdditionalServiceInfoBar.message}
+          touchScreenMessage={
+            this.state.objectsListAdditionalServiceInfoBar.touchScreenMessage
+          }
+        />
+        <InfoBar
+          show={this.state.instancesEditorAdditionalServiceInfoBar.show}
+          identifier={
+            this.state.instancesEditorAdditionalServiceInfoBar.identifier
+          }
+          message={
+            <Trans>
```

Comment:
Trans is not working on variables, this will generate a translation like this `"{0}"`. Only use `Trans` on "real sentences"
Use `i18n._` instead, passing it the result of a `t` macro, which is called a `MessageDescriptor`.

In other words, store in your object the result of `t`:
```
{
  message: t`Hello world`
}
```

then use i18n._ to get the translation as a string.

See how it's done in `Hints/index.js`